atom/renderer/api/lib/remote.js

```diff
@@ -141,7 +141,7 @@ let setObjectPrototype = function(object, metaId, descriptor) {
 };
 
 // Convert meta data from browser into real value.
-var metaToValue = function(meta) {
+let metaToValue = function(meta) {
```

Comment:
(Just attempted to solve #4733).
Is there a reason why this is a `let metaToValue = function